caseworker-api/src/App/src/Spreadsheet/PhpSpreadsheetGenerator.php

```diff
@@ -119,6 +121,30 @@ public function generateFile(
         throw new InvalidArgumentException('Supplied schema and file format is not supported');
     }
 
+
+    function logRowDiagnostics(SpreadsheetRow $row)
+    {
+        $cells=  $row->getCells();
+        $isCheque = false;
+        $referenceNo = "N/A";
+        if (count($cells)> 0) {
+            $rowNumber = cells[0]->getRow();
+
+           // we have to look for column details.
+            foreach ($cells as $cell) {
+                if ($cell->getColumn()==4) {
+                    $referenceNo = $cell->getData();
+                } else {
+                    continue;
+                }
+            }
+            $this->getLogger()->debug("DEBUG-SHEET-OUTPUT: row:" . $rowNumber . " Reference:" . $referenceNo);
+        }else{
+            $this->getLogger()->debug("DEBUG-SHEET-OUTPUT: No cells set for this row.");
+        }
+    }
+
+
```

Comment:
What if anything goes wrong with this? Since this is just a logging function, can we swallow exceptions so it doesn't break anything else?  Or, are we not too bothered because this is a retired service anyway, just being temporarily spun up to get some data out?

Reply:
it is exactly your last point, and if that exception is thrown - we don't swallow it it gets logged later up the stack. i'd rather it break than swallow things.

Reply:
will be getting product to do a test run, so we will find out if this works.

Reply:
throwing this change out, as issue with it. will keep the infra change.